harness/data/haystack.fa

```diff
@@ -0,0 +1 @@
+atgtgatgccatgcacttacaaagaactatccatcagagctaccgaattacgcggagttacgttcggggtcttaatagcatttgaggaaatgtctgcaga
```

Comment:
```suggestion
>my_fasta_1
atgtgatgccatgcacttacaaagaactatccatcagagctaccgaattacgcggagttacgttcggggtcttaatagcatttgaggaaatgtctgcaga
```

switch format to proper fasta format

Reply:
The mock search doesn't understand fasta, but we definitely want this to be a valid file of one sort or another (depending on which formats we decide to support)

Reply:
awesome, thanks!